anvil-python/anvil/commands/haproxy.py

```diff
@@ -34,11 +38,48 @@
 HAPROXY_UNIT_TIMEOUT = (
     1200  # 15 minutes, adding / removing units can take a long time
 )
+HAPROXY_CERTS_DIR = os.path.join(environ['HOME'], "certs/")
+LOG = logging.getLogger(__name__)
+
+
+def validate_cert_file(filepath: str) -> None:
```

Comment:
this seemed unhappy when the path was None, might be worth adding a step ensuring this is a string before using it as a filepath

Reply:
Made None a valid option (in the case where user wants to proceed without TLS)